data/saas/config/adobe_sign_config.yml

```diff
@@ -0,0 +1,108 @@
+    - name: domain
+      default_value: api.na3.adobesign.com
+    - name: client_id
+      label: Client ID
+    - name: client_secret
+      sensitive: True
+    - name: redirect_uri
+      label: Redirect URI
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+      strategy: oauth2_authorization_code
+      configuration:
+        authorization_request:
+          method: GET
+          client_config:
+            protocol: https
+            host: secure.na3.adobesign.com
+          path: /public/oauth/v2
+          query_params:
+            - name: client_id
+              value: <client_id>
+            - name: redirect_uri
+              value: <redirect_uri>
+            - name: response_type
+              value: code
+            - name: scope
+              value: user_read:account user_write:account agreement_read:account agreement_write:account
+            - name: state
+              value: <state>
+        token_request:
+          method: POST
+          path: /oauth/v2/token
+          headers:
+            - name: Content-Type
+              value: application/x-www-form-urlencoded
+          body: |
+            {
+              "client_id": "<client_id>",
+              "client_secret": "<client_secret>",
+              "grant_type": "authorization_code",
+              "code": "<code>",
+              "redirect_uri": "<redirect_uri>"
+            }
+        refresh_request:
+          method: POST
+          path: /oauth/v2/refresh
+          headers:
+            - name: Content-Type
+              value: application/x-www-form-urlencoded
+          body: |
+            {
+              "client_id": "<client_id>",
+              "client_secret": "<client_secret>",
+              "grant_type": "refresh_token",
+              "refresh_token": "<refresh_token>",
+              "redirect_uri": "<redirect_uri>"
+            }
+
+  test_request:
+    method: GET
+    path: /api/rest/v6/users
+
+  endpoints:
+    - name: users
+      requests:
+        read:
+          method: GET
+          path: /api/rest/v6/users
+          param_values:
+            - name: placeholder
+              identity: email
+          data_path: userInfoList
+          postprocessors:
+            - strategy: filter
+              configuration:
+                field: email
+                value:
+                  identity: email
+    - name: agreements
+      requests:
+        read:
+          method: GET
+          path: /api/rest/v6/agreements
+          param_values:
+            - name: placeholder
+              identity: email
+          data_path: userAgreementList
+          postprocessors:
+            - strategy: filter
+              configuration:
+<<<<<<< HEAD
+                field: displayParticipantSetInfos.0.displayUserSetMemberInfos.0.email
+=======
+                field: displayParticipantSetInfos.displayUserSetMemberInfos.email
+>>>>>>> 334d62877ddf3c9597ab2bfbda5f366966ee06fa
```

Comment:
looks like some git merge garbage was committed on accident?

Reply:
yes, i fixed the conflicts in my local but somehow it reverted back and i committed accidentally